transforms/universal/hap/ray/Makefile

```
@@ -0,0 +1,58 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=hap
+
+BASE_IMAGE=${RAY_BASE_IMAGE}
+HAP_PYTHON_VERSION= $(DPK_VERSION)
+
+venv::	.transforms.ray-venv
+
+install:: pip install -r requirements.txt
+
+test::	.transforms.ray-test
+
+clean:: .transforms.clean
+
+image:: .transforms.ray-image
+
+test-src:: .transforms.test-src
+
+setup:: .transforms.setup
+
+test-image:: .transforms.ray-test-image
+
+build:: build-dist image
+
+publish: publish-image
+
+publish-image:: .transforms.publish-image-ray
+
+setup:: .transforms.setup
+
+# distribution versions is the same as image version.
+set-versions:
+	$(MAKE) TRANSFORM_PYTHON_VERSION=$(HAP_PYTHON_VERSION) TOML_VERSION=$(HAP_RAY_VERSION) .transforms.set-versions 
```

```suggestion
	$(MAKE) TRANSFORM_PYTHON_VERSION=$(HAP_PYTHON_VERSION) TOML_VERSION=$(HAP_PYTHON_VERSION) .transforms.set-versions 
```
HAP_RAY_VERSION is not defined. We should use the same version for both PYTHON and RAY

---

tagging @shahrokhDaijavad as he is tracking this closely. @ian-cho One more change that is needed. Please review and accept if you agree. I will do one more pass tomorrow morning.